### ISSUE TITLE
can't run dotnet on 8 gb ram android environment

### ISSUE THREAD

**Issue by ghost** [dotnet/core#8713]
Problem encountered on https://dotnet.microsoft.com/en-us/learn/languages/fsharp-hello-world-tutorial/install
Operating System: linux

Provide details about the problem you're experiencing. Include your operating system version, exact error message, code sample, and anything else that is relevant.

I am developing in termux on android. I installed ubuntu via proot. Then In ubuntu I installed dotnet-sdk-7.0.

I have to install it through the package manager rather than getting the binary.

This is the result when I try dotnet -h

root@localhost:~# dotnet -h
GC heap initialization failed with error 0x8007000E
Failed to create CoreCLR, HRESULT: 0x8007000E
root@localhost:~#

I get the same error when I run dotnet new

I need to be able to run dotnet in this 8 gb android environment even if I need to run it in swapspace/physical memory

**Comment by buyaa-n** [comment]
> I am developing in termux on android. I installed ubuntu via proot. Then In ubuntu I installed dotnet-sdk-7.0.

Might not supported tag @dotnet/linux-team for clarification

**Comment by mairaw** [comment]
@leecow can you help here? is this a supported scenario?

**Comment by leecow** [comment]
No, not specifically supported. @memesmith1, I've never worked on an Android device - what's the processor chipset?

**Comment by memesmith1** [comment]
Don't you reason that it is problematic though for a hello world to take over a gigabyte of ram?

**Comment by memesmith1** [comment]
I was on a Google pixel 7a in an Ubuntu container

**Comment by carlossanlop** [comment]
User deleted their account. I think we can close this as we cannot resolve it further.
